### PR TITLE
Implement CORS

### PR DESCRIPTION
**Pull request by masylum**
Not sure if you had another idea in mind. I just need to be able to set `withCredentials`.

### CODE REVIEW

**Comment by albburtsev** [comment]
Looks good. Could you update JSDoc about `cors` flag and add JSDoc about `[settings.cors]`?

**Comment by masylum** [comment]
Done :smile:

**Comment by masylum** [comment]
I can add tests later on too.

File: src/fetch-please.js [hunk 1]
```diff
@@ -54,12 +55,13 @@ class FetchPlease {
         assign(this, {
             timeout: 0,
             headers: {},
+            cors: false
```
Comment (andresgutgon) [inline src/fetch-please.js:58]: Typo you're missing a `,`

**Review by andresgutgon — CHANGES_REQUESTED** [review]

File: src/fetch-please.js (hunk 1 above)
Comment (JulianWielga) [reply src/fetch-please.js:58]: this pr looks dead so i fixed this typo in #8

**Comment by albburtsev** [comment]
Merged in #8.

**Pull request closed without merging**